db/migrate/20161126190339_add_draft_to_events.rb

```diff
@@ -1,5 +0,0 @@
-class AddDraftToEvents < ActiveRecord::Migration
-  def change
-    add_column :events, :draft, :boolean
-  end
-end
```

Comment:
[This page](http://edgeguides.rubyonrails.org/active_record_migrations.html#running-migrations) has everything you need to know about migrations. I really recommend to read it through.